atc/validate.go

```diff
@@ -217,13 +217,33 @@ func validateJobs(c Config) ([]ConfigWarning, error) {
 			errorMessages = append(errorMessages, identifier+" has no name")
 		}
 
-		if job.BuildLogsToRetain < 0 {
+		if job.BuildLogRetention != nil && job.BuildLogsToRetain != 0 {
+			errorMessages = append(
+				errorMessages,
+				identifier+fmt.Sprintf(" can't use both build_log_retention and deprecated build_logs_to_retain"),
```

Comment:
```suggestion
				identifier+fmt.Sprintf(" can't use both build_log_retention and build_logs_to_retain"),
```

We haven't deprecated it just yet - the validation makes sense but we haven't decided whether we want to deprecate the shorthand form. :slightly_smiling_face: